sql/spanner/migrate.go

```diff
@@ -298,7 +366,8 @@ func check(b *sqlx.Builder, c *schema.Check) {
 }
 
 // Build instantiates a new builder and writes the given phrase to it.
-func Build(phrase string) *sqlx.Builder {
-	b := &sqlx.Builder{QuoteChar: '`'}
-	return b.P(phrase)
+func (s *state) Build(phrases ...string) *sqlx.Builder {
+	emptySchema := ""
+	b := &sqlx.Builder{QuoteChar: '`', Schema: &emptySchema}
```

Comment:
```suggestion
	b := &sqlx.Builder{QuoteChar: '`', Schema: sqlx.P("")}
```
https://github.com/ariga/atlas/blob/master/sql/internal/sqlx/sqlx.go#L467